Documentation/style-guide.md

```diff
@@ -70,6 +74,33 @@ Instead of
 import Header from "../../../../../../components/Header/Header";
 ```
 
+#### Exceptions (Siblings and Children)
+
+As explained above, we wan to use relative imports when it comes to importing sibling files, or children. For example.
+
+```tsx
+import styles from "./Header.module.scss";
+```
+Or
+
+```tsx
+import Calender from "./components/Calender/Calender";
```

Comment:
Maybe add "When importing subcomponents" or something like that, and this is just a suggestion to make it more clear that it is just a part of something and not a component that necessarily make sense alone
```suggestion
import CalenderHeader from "./components/CalenderHeader/CalenderHeader";
```